python/packages/isce3/math/offsets_polyfit.py

```diff
@@ -0,0 +1,508 @@
+import numpy as np
+
+def ncoeffs(degree: int) -> int:
+    """
+    Compute the number of coefficients in a 2D polynomial
+    with total degree less than or equal to a given value.
+
+    Parameters
+    ----------
+    degree : int
+        The maximum total degree of the 2D polynomial.
+
+    Returns
+    -------
+    int
+        The number of unique coefficients in the polynomial.
+    """
+    return (degree + 1) * (degree + 2) // 2
+
+
+def normalize(x, xmin, xmax):
```

Comment:
```suggestion
def normalize_to_unit_range(x, xmin=None, xmax=None):
```
This name is too generic for the `isce3.math` namespace. Also it would be more useful if the function could compute the extrema itself, though I can see why you might want the option not to.